src/api/supabase/queries/profiles.ts

```diff
@@ -130,7 +135,10 @@ export async function deleteRoles(userId: UUID, deleteInfo: ProfileRole[]) {
     .from('profiles-roles')
     .delete()
     .eq('user_id', userId)
-    .in('role', Array.from(new Set(deleteInfo.map(i => i.role))));
+    .in(
```

Comment:
What does .in() do again?

Reply:
it's a filter like `.eq`, basically filters only rows whose value for `role` is within the specified array
for more information, see [official documentations](https://supabase.com/docs/reference/javascript/in)